openhantek/src/settings.h

```diff
@@ -21,6 +21,7 @@ class DsoSettings {
 
     DsoSettingsExport exporting; ///< General options of the program
     DsoSettingsScope scope;     ///< All oscilloscope related settings
+    const Dso::ControlSpecification* deviceSpecification; ///< Device specification settings
```

Comment:
That slipped in I guess? I have removed the `deviceSpecification` again. It is always accessible via `DsoControl->getDevice()->getModel().spec`

Reply:
Yep, i couldn't find it. I will remove it!